encrypted-media-respec.html

```diff
@@ -1598,6 +1591,10 @@
         The above rule implies that the CDM instance must not be destroyed until all <a>MediaKeys</a>
         objects and all <a>MediaKeySession</a> objects associated with the CDM instance are destroyed.
       </p>
+      <p>
+        When a <a>MediaKeySession</a> object is destroyed, the CDM SHALL close the <a href="#key-session">key session</a> associated with the object.
```

Comment:
The terminology 'when object X is destroyed' often gets interpreted in an implementation-dependent way, for example, the time at which garbage collection destroys some in-memory data structure. Should we use implementation-dependent language in our specification ?

The previous requirement was deliberately based on an implementation-independent condition.